Java/QuickSort.java

```diff
@@ -0,0 +1,47 @@
+public class QuickSort {
```

Comment:
update java folder readme

Reply:
I just updated the java folder README.md!